### ISSUE TITLE
'BroadcastView' has no propType for native prop 'RNBroadcastView.onFrameChange' of native type 'BOOL'

### ISSUE THREAD

**Issue by RazvanCosmeanu** [medlmobileenterprises/react-native-wowza-gocoder#4]
Thanks for sharing this life saving library, guys. Trying to get the example project to work, but I get this when trying to run on a device. Thanks!

```
`BroadcastView` has no propType for native prop `RNBroadcastView.onFrameChange` 
of native type `BOOL`
If you haven't changed this prop yourself, this usually means that your versions
of the native code and JavaScript code are out of sync. 
Updating both should make this error go away.
```

**Comment by RazvanCosmeanu** [comment]
Managed to get it to work by removing the following lines from RNBroadcastViewManager.h

`RCT_EXPORT_VIEW_PROPERTY(onFrameChange, RCTDirectEventBlock);`

and remove `@"onFrameChange"` from `customDirectEventTypes` array

**Comment by hugo-ns** [comment]
Thanks for the feedback.
We recently remove this props, in order to improve the consistency between both iOS/Android platforms. 
It seems like we left unused code. You're welcome to pull request your changes. We'll include it in the next release.

**Comment by hugo-ns** [comment]
Fix added to release 1.0.3
